journalism_workflow/static/journalism_workflow/photographer/partials/photographer.html

```diff
@@ -0,0 +1,80 @@
+<section class="section-panel">
+  <div class="container-fluid">
+    <div class="row section-header">
+      <div class="col-lg-12 col-md-12 col-sm-12">
+        <h3>
+	  Take and upload photos for the article
+        </h3>
+      </div>
+    </div>
+
+    <!-- Titles for the two columns -->
+    <div class="row padded">
+      <div class="col-lg4 col-md-4 col-sm-4">
+	<h3>Editor's Story Plan</h3>
+      </div>
+
+      <div class="col-lg-8 col-md-8 col-sm-8">
+	<h3>Photographer Instructions</h3>
+      </div>
+
+    </div>
+
+    <div class="row padded">
+
+      <!-- Form with Editor's story plan -->
```

Comment:
+1 for cleanup, especially if @derek-schultz is going to improve our angular organization.

Do we have a wishlist for that issue, or should we do some brainstorming before creating it?